packages/vue/src/components/organisms/SfProductCardHorizontal/SfProductCardHorizontal.vue

```diff
@@ -0,0 +1,329 @@
+          :is="linkComponentTag"
+          :href="linkComponentTag === 'a' ? link : undefined"
+          :to="link && linkComponentTag !== 'a' ? link : undefined"
+          class="sf-product-card-horizontal__link"
+        >
+          <slot name="title" v-bind="{ title }">
+            <h3 class="sf-product-card-horizontal__title">
+              {{ title }}
+            </h3>
+          </slot>
+        </component>
+        <slot name="price" v-bind="{ specialPrice, regularPrice }">
+          <SfPrice
+            v-if="regularPrice"
+            class="sf-product-card-horizontal__price"
+            :regular="regularPrice"
+            :special="specialPrice"
+          />
+        </slot>
+      </div>
+      <button
+        v-if="wishlistIcon !== false"
+        :aria-label="ariaLabel"
+        :class="wishlistIconClasses"
+        class="mobile-only"
+        @click="toggleIsOnWishlist"
+      >
+        <slot name="wishlist-icon" v-bind="{ currentWishlistIcon }">
+          <SfIcon
+            :icon="currentWishlistIcon"
+            color="black"
+            size="19px"
+            data-test="sf-wishlist-icon"
+          />
+        </slot>
+      </button>
+      <div class="sf-product-card-horizontal__details">
+        <slot name="description" v-bind="{ description }">
+          <p class="sf-product-card-horizontal__description desktop-only">
+            {{ description }}
+          </p>
+        </slot>
+        <slot name="options" v-bind="{ options }">
+          <div class="sf-product-card-horizontal__options">
+            <SfProperty
+              v-for="(option, i) in options"
+              :key="i"
+              :name="option.name"
+              :value="option.value"
+              class="sf-product-card-horizontal__property"
+            />
+          </div>
+        </slot>
+        <slot name="reviews" v-bind="{ maxRating, scoreRating }">
+          <div
+            v-if="typeof scoreRating === 'number'"
+            class="sf-product-card-horizontal__reviews"
+          >
+            <SfRating
+              class="sf-product-card-horizontal__rating"
+              :max="maxRating"
+              :score="scoreRating"
+            />
+            <a
+              v-if="reviewsCount"
+              class="sf-product-card-horizontal__reviews-count"
+              href="#"
+              @click="$emit('click:reviews')"
+            >
+              {{ reviewsCount }} reviews
+            </a>
+          </div>
+        </slot>
+      </div>
+      <slot name="actions">
+        <div class="sf-product-card-horizontal__actions desktop-only">
+          <SfButton
+            class="sf-button--text"
+            @click="$emit('click:add-to-wishlist')"
+          >
+            Save for later
+          </SfButton>
+          <SfButton
+            class="sf-button--text"
+            @click="$emit('click:add-to-compare')"
+          >
+            Add to compare
+          </SfButton>
+          <SfAddToCart
```

Comment:
IMO SfAddToCart shouldn't be inside slot "actions". Wha you think to create for it special slot "add-to-cart" like on ProductCard

Reply:
👍